specs/_features/maxeb_increase/capella.py

```diff
@@ -1627,14 +1646,37 @@ def process_pending_balance_deposits(state: BeaconState) -> None:
     state.pending_balance_deposits = state.pending_balance_deposits[next_pending_deposit_index:]
 
 
+def apply_pending_consolidation(state: BeaconState, pending_consolidation: PendingConsolidation) -> None:
+    source_validator = state.validators[pending_consolidation.source_index]
+    target_validator = state.validators[pending_consolidation.target_index]
+    # Move active balance
```

Comment:
I think we should abort consolidation if either the source or the target is slashed